### PR TITLE
Added arguments usage example

### PR DESCRIPTION
**Pull request by v0idpwn**
You could find something similar on the code but it was not clear. I had to use this and maybe it could help someone.

### CODE REVIEW

File: lib/app/commands.ex [hunk 1]
```diff
@@ -87,6 +87,13 @@ defmodule App.Commands do
     ]
   end
 
+  # You can emulate arguments acess through nadia's update.message
```
Comment (lubien) [inline lib/app/commands.ex]: Typo "acess".

File: lib/app/commands.ex [hunk 2]
```diff
@@ -87,6 +87,13 @@ defmodule App.Commands do
...
+  command "argued" do
+    Logger.log :info, "Command /argued"
+     [_command | args] = String.split(update.message.text, " ")
```
Comment (lubien) [inline lib/app/commands.ex]: Wrong indent

**Review by lubien — CHANGES_REQUESTED** [review]

**Comment by lubien** [comment]
Hi there. It's something really useful for sure. Just need some fixes before merging.

**Comment by v0idpwn** [comment]
fix'd. 

create more issues, i'm willing to help :)

**Review by lubien — APPROVED** [review]

**Comment by lubien** [comment]
Lovely. Thank you.

Also sorry for the delay

**Pull request merged**